### PR TITLE
Version 1.2.0

### PR DESCRIPTION
**Pull request by sdruskat**
Updates to go into version 1.2.0.

### CODE REVIEW

File: schema-sandbox/schema.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        "abstract": {
+            "type": "string"
+        },
+        "authors": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
+                "anyOf": [
+                    {"$ref": "#definitions/person"},
+                    {"$ref": "#definitions/entity"}
```
Comment (jspaaks) [inline schema-sandbox/schema.json:34]:
```suggestion
                    {"$ref": "#/definitions/person"},
                    {"$ref": "#/definitions/entity"}
```
probably needs slashes

File: schema-sandbox/schema.json [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                "software"
+            ]
+        },
+        "abstract": {
+            "type": "string"
+        },
+        "authors": {
+            "type": "array",
+            "minItems": 1,
+            "items": {
```
Comment (jspaaks) [inline schema-sandbox/schema.json:31]:
```suggestion
            "minItems": 1,
            "uniqueItems": true,
            "items": {
```

File: schema-sandbox/schema.json (hunk 1 above)
Comment (sdruskat) [reply schema-sandbox/schema.json:35]: Does without, but if it works with them, I'd prefer it with them... I'll try soon as I can.

**Comment by sdruskat** [comment]
Merging as discussed during a review call.

**Pull request merged**